### PR TITLE
Create publishing bot issue

### PR DESCRIPTION
**Pull request by hoegaarden**
When we create a new branch, we can also create an issue for publishing-bot folks, asking them to update the bot's configuration. Also see [this convo in slack](https://kubernetes.slack.com/archives/C2C40FMNF/p1551439046077800?thread_ts=1551438459.077100&cid=C2C40FMNF).

/cc @sttts @nikhita @dims
/assign @kubernetes/bash-firefighters 
/area release-eng

### CODE REVIEW

File: lib/gitlib.sh [hunk 1]
```diff
@@ -307,20 +307,65 @@ if ((FLAGS_gcb)); then
...
+  local repo title body assign milestone
+
+  title="Update publishing-bot for ${branch}"
+  repo="k8s-release-robot/sig-release"
+  assign="<!-- no assignments -->"
+  milestone="v${branch#release-}"
+
+  if ((FLAGS_nomock)); then
+    repo="kubernetes/kubernetes"
+    assign="/assign @sttts @nikhita @dims"
```
Comment (hoegaarden) [inline lib/gitlib.sh]: I am not sure if we'd want to assign specific people or if we have some GH team or such?

Comment (nikhita) [reply lib/gitlib.sh]: I'd prefer a GH team over specific people.

File: lib/gitlib.sh [hunk 2]
```diff
@@ -332,3 +332,40 @@ gitlib::create_issue() {
...
+Please update the [publishing-bot's configuration][config] to also publish this
+new branch.
+
+[branch]: https://github.com/kubernetes/kubernetes/tree/${branch}
+[config]: https://github.com/kubernetes/kubernetes/tree/master/staging/publishing
+
+${assign}
+/sig release
+/area release-eng
+/area code-generation
```
Comment (nikhita) [inline lib/gitlib.sh]: The `area/code-generation` label is used mainly for the k/code-generator repo related issues, so I'd keep only the `/area release-eng` command here.

File: lib/gitlib.sh (hunk 1 above)
Comment (nikhita) [reply lib/gitlib.sh]: Created https://github.com/kubernetes/org/pull/558

Comment (hoegaarden) [reply lib/gitlib.sh]: It seems `/assign`ing to an GH team does not work.
I tried to assign this very PR to `@kubernetes/bash-firefighters` but the @k8s-ci-robot didn't pick that up.

Comment (nikhita) [reply lib/gitlib.sh]: Just tried using the UI, looks like GH doesn't support assigning teams. :-1: 

I'd be fine with cc'ing a team as well. But if others feel that assigning is more powerful, I'm ok with assigning specific people for now. Overall, just want to ensure that we don't create a SPOF-ish situation. :)

@sttts @dims thoughts?

Comment (hoegaarden) [reply lib/gitlib.sh]: Just before we create the issue we could ask the GH API who is on that team and assign to them. I'll look into that.

Comment (hoegaarden) [reply lib/gitlib.sh]: 🐚 🔥 😼

**Comment by nikhita** [comment]
/cc @BenTheElder 
for graphql stuff

**Comment by BenTheElder** [comment]
first take: this is amazing!
second take: anago got bigger 😬

**Comment by hoegaarden** [comment]
> I just have some broader sneaking concerns about anago :see_no_evil:

You are not alone on that.

**Comment by nikhita** [comment]
:wave: bump

**Comment by nikhita** [comment]
> This looks well implemented FWIW, I just have some broader sneaking concerns about anago 

@BenTheElder do you think this can be merged or something else needs to be done here?

**Comment by BenTheElder** [comment]
/shrug
I won't block it but I think we should really consider writing greenfield automation outside of the anago scripts.

**Pull request merged**